### PR TITLE
Update various templates to use status-icon

### PR DESCRIPTION
**Pull request by benjaminapetersen**
@jwforres @spadgett this is pt2 of [breaking down this PR](https://github.com/openshift/origin/pull/5953)

### CODE REVIEW

File: assets/app/views/browse/_build-details.html [hunk 1]
```diff
@@ -38,7 +31,7 @@
           </span>
         </dd>
       </dl>
-      <h3>Configuration <span class="small" ng-if="buildConfigName">created from <a href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfigName}}">{{buildConfigName}}</a></span></h3>        
+      <h3>Configuration <span class="small" ng-if="buildConfigName">created from <a href="project/{{build.metadata.namespace}}/browse/builds/{{buildConfigName}}">{{buildConfigName}}</a></span></h3>
```
Comment (jwforres) [inline assets/app/views/browse/_build-details.html]: guessing you created a merge conflict with your other PR here, you can drop the whitespace fix

Comment (jwforres) [inline assets/app/views/browse/_build-details.html]: or wait and rebase after your other one merges, but who knows when that will actually go through :)

File: assets/app/views/browse/build-config.html [hunk 2]
```diff
@@ -182,22 +182,14 @@
...
-                <!-- Build status -->
-                <span ng-switch="build.status.phase" class="hide-ng-leave status-icon">
-                  <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="Error" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
-                  <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-default class="fa fa-refresh" aria-hidden="true"></span>
-                </span>
+                <status-icon status="build.status.phase"></status-icon>
```
Comment (jwforres) [inline assets/app/views/browse/build-config.html]: disable animation

File: assets/app/views/browse/deployment-config.html [hunk 3]
```diff
@@ -118,18 +118,9 @@
...
-                <span ng-switch="deployment | deploymentStatus" class="hide-ng-leave status-icon">
-                  <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>              
-                  <span ng-switch-when="Deployed" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true"></span>
-                  <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true"></span>
-                  <span ng-switch-when="New" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-when="Pending" class="spinner spinner-xs spinner-inline" aria-hidden="true"></span>
-                  <span ng-switch-default class="fa fa-refresh" aria-hidden="true"></span>
-                </span>
+                <status-icon status="deployment | deploymentStatus"></status-icon>
```
Comment (jwforres) [inline assets/app/views/browse/deployment-config.html]: disable animation

File: assets/app/views/browse/_build-details.html (hunk 1 above)
Comment (benjaminapetersen) [inline assets/app/views/browse/_build-details.html]: My editor does it automatically... I'll have to live with it I suppose unless I disable the feature.  One day we won't have those issues anymore :smile:

**Comment by benjaminapetersen** [comment]
[TEST]

**Comment by benjaminapetersen** [comment]
@jwforres updated the comments

**Comment by jwforres** [comment]
pretty sure this will end up with a merge conflict, but what the heck [merge] hopefully at least one of them passes and gets in :)

**Comment by benjaminapetersen** [comment]
[TEST] rebased

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Pull request merged**